internal/trigger/buildkite/buildkite.go

```diff
@@ -94,6 +94,9 @@ func (t *Trigger) Do(_ []byte) error {
 	type outputData struct {
 		Queue              string `json:"queue"`
 		ScheduledJobsCount int    `json:"scheduled_jobs_count"`
+		RunningJobsCount   int    `json:"running_jobs_count"`
+		WaitingJobsCount   int    `json:"waiting_jobs_count"`
+		TotalJobsCount     int    `json:"total_jobs_count"`
```

Comment:
We will need to make use of these here: https://github.com/scriptnull/waymond/blob/b552d53f85affc019593229b97fa6f92883f4481/internal/trigger/buildkite/buildkite.go#L112-L113